frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -775,6 +778,24 @@ function FeatureFlagRollout({ readOnly }: { readOnly?: boolean }): JSX.Element {
         })
     }
 
+    const confirmEncryptedPayloadReset = (): void => {
+        LemonDialog.open({
+            title: 'Reset payload?',
+            description: 'The existing payload will not be deleted until the feature flag is saved.',
```

Comment:
```suggestion
            title: 'Reset payload?',
            description: 'The existing payload will not be reset until the feature flag is saved.',
```

is it `deleted`, or `reset`?  You say one thing in the title another in the description, I'd opt for using reset everywhere.

Reply:
True, sg!